cache/fast_lru_cache_test.cc

```diff
@@ -0,0 +1,73 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
I'm not a fan of new test binaries like this, because if every little feature had its own test binary, we would have quadratic blow-up in build size and build time (at least in default static linking build). Perhaps you want to minimize pollution of cache_test with concerns of our non-public prototyping efforts in FastLRUCache. My preference would be just naming the test class FastLRUCacheTest in cache_test.

Reply:
Moved to lru_cache_test.cc.